src/main/resources/lang/default.lang

```diff
@@ -2235,7 +2237,40 @@ attribute types:
 	player.sneaking_speed: player sneaking speed, sneaking speed
 	player.submerged_mining_speed: player submerged mining speed, submerged mining speed
 	player.sweeping_damage_ratio: player sweeping damage ratio, sweeping damage ratio
-	zombie.spawn_reinforcements: zombie spawn reinforcements
+	zombie.spawn_reinforcements: zombie spawn reinforcementsgeneric.armor: generic armor, armor
+	# 1.21.3 removes prefixes, adds armor and tempt range
+	armor: generic armor, armor
+	armor_toughness: generic armor toughness, armor toughness
+	attack_damage: generic attack damage, attack damage
+	attack_knockback: generic attack knockback, attack knockback
+	attack_speed: generic attack speed, attack speed
+	burning_time: generic burning time, burning time
+	explosion_knockback_resistance: generic explosion knockback resistance, explosion knockback resistance
+	flying_speed: generic flying speed, flying speed
+	follow_range: generic follow range, follow range
+	gravity: generic gravity, gravity
+	jump_strength: generic jump strength, jump strength, horse jump strength
+	knockback_resistance: generic knockback resistance, knockback resistance
+	luck: generic luck, luck
+	max_absorption: generic max absorption, max absorption
+	max_health: generic max health, max health
+	movement_efficiency: generic movement efficiency, movement efficiency
+	movement_speed: generic movement speed, movement speed
+	oxygen_bonus: generic oxygen bonus, oxygen bonus
+	safe_fall_distance: generic safe fall distance, safe fall distance
+	fall_damage_multiplier: generic fall damage multiplier, fall damage multiplier
+	scale: generic scale, scale
+	step_height: generic step height, step height
+	tempt_range: generic tempt range, tempt range
+	water_movement_efficiency: generic water movement efficiency, water movement efficiency
+	block_break_speed: player block break speed, block break speed
+	block_interaction_range: player block interaction range, block interaction range
+	entity_interaction_range: player entity interaction range, entity interaction range
+	mining_efficiency: player mining efficiency, mining efficiency
+	sneaking_speed: player sneaking speed, sneaking speed
+	submerged_mining_speed: player submerged mining speed, submerged mining speed
```

Comment:
maybe add underwater as an option? or should it be as close to the original as possible

Reply:
this is just for 1.21.3, so this is out of scope